components/log-viewer-webui/package.json

```diff
@@ -0,0 +1,22 @@
+{

```

Comment:
Shall we add the `package-lock.json` file to VCS?

Reply:
Sorry, thought I added it.